pupil-api/src/services/redis-pupil-auth.service.spec.ts

```diff
@@ -65,10 +67,14 @@ describe('redis-pupil-auth.service', () => {
   })
 
   test('the check payload should be returned if item found in cache', async () => {
```

Comment:
This test, _as described_, is not a valid test, as the pin must pass time-based validity checks.